release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1441,6 +1441,10 @@ sourceStrategy:
 
 * Previously, when destroying a partially deployed cluster after an installation failed on VMWare vSphere, some virtual machine folders were not destroyed. This error could occur in clusters configured with multiple vSphere datacenters or multiple vSphere clusters. With this update, all installer-provisioned infrastructure is correctly deleted when destroying a partially deployed cluster after an installation failure. (link:https://issues.redhat.com/browse/OCPBUGS-1489[*OCPBUGS-1489*])
 
+* Previously, when installing a cluster on VMWare vSphere, the installation failed if the user specified the `platform.vsphere.vcenters` parameter but did not specify the `platform.vsphere.failureDomains.topology.networks` parameter in the `install-config.yaml` file. With this update, the installation program alerts the user that the `platform.vsphere.failureDomains.topology.networks` field is required when specifying `platform.vsphere.vcenters`. (link:https://issues.redhat.com/browse/OCPBUGS-1698[*OCPBUGS-1698*])
+
+* Previously, when installing a cluster on VMWare vSphere, the installation would fail if the user defined the `platform.vsphere.vcenters` and `platform.vsphere.failureDomains` parameters but did not define `platform.vsphere.defaultMachinePlatform.zones`, or `compute.platform.vsphere.zones` and `controlPlane.platform.vsphere.zones`. With this update, the installation program validates that the user has defined the `zones` parameter in multi-region or multi-zone deployments prior to installation. (link:https://issues.redhat.com/browse/OCPBUGS-1490[*OCPBUGS-1490*])
```

Comment:
Super picky: the  first entry uses _installation failed_ where the second uses _installation would fail_. 
Feel free to ignore!